client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -181,8 +183,17 @@ class CalypsoifyIframe extends Component<
 		}
 	}
 
+	componentDidUpdate( prevProps: ConnectedProps ) {
+		const mediaGutenframeActions = this.props.mediaGutenframeActions;
+		if (
+			mediaGutenframeActions !== prevProps.mediaGutenframeActions &&
```

Comment:
Is `!==` always enough, given that two separate arrays with the same contents will be different when strictly compared?